setup.py

```diff
@@ -1,39 +1,54 @@
-from setuptools import setup, find_packages
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from setuptools import find_packages, setup
 
-version = __import__('sekizai').__version__
+from sekizai import __version__
+
+
+REQUIREMENTS = [
+    'django>=1.11',
+    'django-classy-tags>=0.3.1',
```

Comment:
Perhaps this needs to be bumped - classy tags changelog says 0.8.0 added support for Django 1.10, and 0.9.0 added testing for 1.11